src/settings/toml/site.rs

```diff
@@ -8,7 +8,7 @@ use crate::commands::generate::run_generate;
 
 const SITE_ENTRY_POINT: &str = "workers-site";
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
```

Comment:
added to allow comparison of Manifests for the benefit of testing constructor methods against one another.